### PR TITLE
Xenial stable tag updates

### PR DESCRIPTION
**Pull request by MariadeAnton**

### CODE REVIEW

File: user/build-environment-updates/2019-01-10.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+# 2019-01-10
+
+## Changes
+- Language: generic is recognized
+- System Info is more readable
+- PhantomJS2 is available
+- Tmate is preinstalled
+- Gem (and rvm) are configured to use `--no-document` rather than `--no-ri` and `--no-rdoc`.
+- Ensure ip forwarding on localhost does not get interrupted once per day
```
Comment (MariadeAnton) [inline user/build-environment-updates/2019-01-10.md]: would it be good for context to link to the public issue / forum post about this?

Comment (plaindocs) [reply user/build-environment-updates/2019-01-10.md]: I'm +1 on that

File: user/build-environment-updates/2019-01-10.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- Openjdk certificates use system certificates
+- Gradle daemon is not disabled anymore
+- Only set `_JAVA_OPTIONS` on docker containers
+- Drop hhvm support
+- Chromium is not installed anymore (Google Chrome, however, continues to be installed)
+- Xvfb init file is removed in favor of a systemd unit file. Xvfb can be set up with `services: xvfb`. The matching DISPLAY environment variable is made available when you do that.
+
+## Version bumps
```
Comment (plaindocs) [inline user/build-environment-updates/2019-01-10.md]:
```suggestion
## Version changes
```

Comment (plaindocs) [inline user/build-environment-updates/2019-01-10.md]:
```suggestion
## Version updates
```

Comment (plaindocs) [reply user/build-environment-updates/2019-01-10.md]: Pick one :-p

**Comment by LLongshore** [comment]
😀

**Pull request merged**